docs/content/get-started/set-up.md

```diff
@@ -140,38 +142,39 @@ If you need help using the GDK, come and talk to us about the software and the d
 <br/>
 
 
-## Get the GDK and Starter Project Source Code
+## Get the GDK and the FPS Starter Project source code
+To run the GDK and the FPS Starter project, you need to download the source code. There are two ways you can do this: _either_ get both sets of source code as one zip file download _or_ clone the two repositories separately using Git. (To find out more about Git, see [github.io](https://try.github.io)).
 
-### Zip Download
+**NOTE:** We recommend using Git, as Git's version control makes it easier for you to get updates in the future.
 
-<%(Callout type="alert" message="
-If you can, please use Git to get the GDK and its starter project, as version control will make it easier for you to get updates in the future.")%>
+### Zip file download
 
-If you are unfamiliar with Git, you can download a Zip file of the GDK and its Starter Project <a href="https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3" data-track-link="Starter Project Zip Clicked|product=Docs|platform=Mac|label=Mac" target="_blank">here</a>. 
+While we recommend using Git, if you prefer to, you can get the source code for both the GDK and FPS Starter Project by downloading one zip file <a href="https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3" data-track-link="Starter Project Zip Clicked|product=Docs|platform=Mac|label=Mac" target="_blank">here</a>. 
```

Comment:
`platform=Mac|label=Mac` is incorrect here as this link is not in the expandable Mac section of the setup guide.

```suggestion
 While we recommend using Git, if you prefer to, you can get the source code for both the GDK and FPS Starter Project by downloading one zip file <a href="https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3" data-track-link="Starter Project Zip Clicked|product=Docs" target="_blank">here</a>. 
```